nano/node/bootstrap.cpp

```diff
@@ -1394,16 +1394,16 @@ void nano::bootstrap_attempt::lazy_add (nano::block_hash const & hash_a)
 	}
 }
 
-void nano::bootstrap_attempt::lazy_pull_flush ()
+void nano::bootstrap_attempt::lazy_pull_flush (std::unique_lock<std::mutex> & lock_a)
```

Comment:
Should something use the lock passed in?

Reply:
Good point, it can be just assert